README.md

```diff
@@ -1,5 +1,9 @@
 # CyberArk Sidecar Injector
 
+Sidecars are used in Kubernetes to introduce additional features to application pods. Manual sidecar injection can be cumbersome and repetitive. **CyberArk Sidecar Injector** enables automatic sidecar injection through it being a [mutating admission webhook controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook). This means when **CyberArk Sidecar Injector** is deployed and enabled in a namespace, any pod created in that namespace, with annotations that indicate a request for injection, will result in automated sidecar injection. **CyberArk Sidecar Injector** provides a selection of available sidecars that are configurable through annotations.
```

Comment:
```suggestion
Sidecars are used in Kubernetes to introduce additional features to application pods. Manual sidecar injection can be cumbersome and repetitive. **CyberArk Sidecar Injector** enables automatic sidecar injection through it being a [mutating admission webhook controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook). This means that when **CyberArk Sidecar Injector** is deployed and enabled in a namespace, any pod created in that namespace with annotations that indicate a request for injection will result in automated sidecar injection. **CyberArk Sidecar Injector** provides a selection of available sidecars that are configurable through annotations with <insert names> being the currently-supported ones.
```

Reply:
non-blocking nit: This paragraph for sure (and maybe even other ones in this doc) may need word wrapping.